muted-tests.yml

```diff
@@ -236,6 +236,9 @@ tests:
 - class: org.elasticsearch.packaging.test.DockerTests
   method: test012SecurityCanBeDisabled
   issue: https://github.com/elastic/elasticsearch/issues/116636
+- class: org.elasticsearch.index.shard.StoreRecoveryTests
```

Comment:
bad merge? same as the other ones that are added?

Reply:
🤦 ouch. Sorry about that.

Opened https://github.com/elastic/elasticsearch/pull/130523 to fix